src/main/java/com/rexdelf/issuetrackerapp/dto/TicketDTO.java

```diff
@@ -0,0 +1,20 @@
+package com.rexdelf.issuetrackerapp.dto;
+
+import lombok.Data;
+
+import java.time.LocalDateTime;
+
+@Data
+public class TicketDTO {
+
+  private Long id;
+  private String title;
+  private String description;
+  private String reporter_name;
```

Comment:
We should follow common java name conventions. Snake case is not appropriate for non-constant variables